source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,103 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 :depth: 2
 ```
 
+## Release v9.3 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release Date: December 15, 2023**
+
+### Important Upgrade Notes
+ - Please read the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) before upgrading.
+
+### Compatibility
+ - Updated minimum required Firefox version to v115+.
+ - Updated minimum supported Chromium version to 118+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated pre-packaged Playbooks plugin version to [v1.39.1](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.39.1).
+ - Updated pre-packaged Calls version to [v0.21.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.21.1).
+ - Updated pre-packaged Jira plugin version to [v4.0.1](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.0.1). Also see [v4.0.0](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.0.0) for recent breaking changes.
+ - Added Vietnamese (Beta) as a new language.
+ - Added the ability to passively track keywords with highlights without triggering a notification.
+ - Updated the **Settings** modal with an improved user interface.
+ - Added a new **Jump to recents** banner when a channel is scrolled up.
+ - Modified the behavior of the code button (Ctrl+Alt+C) to create inline codes or code blocks.
+ - Disabled markdown keybindings within code blocks.
+ - Added a **Back** button to the ``/access_problem`` page.
+ - Added a default limit of the number of reactions per post.
+
+#### Performance
+ - Removed pre-fetch preference and set new prefetch limits for the webapp.
+ - Improved websocket event marshaling performance.
+ - Batched loading of recently used emojis on initial load.
+
+#### Administration
+ - The tooltip on the announcement bar in the **System Console** is now widened.
+ - Improved the error message when trying to activate a plugin in an unsupported environment.
+ - Added a file storage permission check to the workspace health dashboard.
+ - Performed a cleanup in preparation for adding support for multi-word mentions.
```

Comment:
Performed a cleanup in preparation for adding support for multi-word keywords that trigger notification.